pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -64,28 +66,31 @@ type LaunchTemplate struct {
 type Provider struct {
 	sync.Mutex
 	ec2api                  ec2iface.EC2API
+	eksapi                  eksiface.EKSAPI
 	amiFamily               *amifamily.Resolver
 	securityGroupProvider   *securitygroup.Provider
 	subnetProvider          *subnet.Provider
 	instanceProfileProvider *instanceprofile.Provider
 	cache                   *cache.Cache
-	caBundle                *string
 	cm                      *pretty.ChangeMonitor
 	KubeDNSIP               net.IP
+	CABundle                *string
 	ClusterEndpoint         string
+	ClusterCIDR             *string
```

Comment:
This value is not atomic but is having Read/Write access through different threads. That's going to lead to race conditions. Consider making this an atomic string